ui/src/shared/utils/TimeMachineContainer.ts

```diff
@@ -70,12 +70,13 @@ import {ColorString, ColorNumber} from 'src/types/colors'
 
 const LOCAL_STORAGE_DELAY_MS = 1000
 
-const DEFAULT_STATE = () => ({
+const DEFAULT_STATE = (): TimeMachineState => ({
```

Comment:
hrm is this typing doing anything?

Reply:
it's just there so that if someone changes the TimeMachineState, that they'll be prompted to give it a default state as well